sys-cluster/glusterfs/glusterfs-7.9.ebuild

```diff
@@ -0,0 +1,211 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit autotools elisp-common python-single-r1 systemd
+
+if [[ ${PV#9999} != ${PV} ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/gluster/glusterfs.git"
+else
+	SRC_URI="https://download.gluster.org/pub/gluster/${PN}/$(ver_cut 1)/${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+fi
+
+DESCRIPTION="GlusterFS is a powerful network/cluster filesystem"
+HOMEPAGE="https://www.gluster.org/ https://github.com/gluster/glusterfs/"
+
+LICENSE="|| ( GPL-2 LGPL-3+ )"
+SLOT="0/${PV%%.*}"
+IUSE="debug emacs +fuse +georeplication infiniband ipv6 libressl +libtirpc rsyslog static-libs +syslog test +xml"
+
+REQUIRED_USE="georeplication? ( ${PYTHON_REQUIRED_USE} )
+	ipv6? ( libtirpc )"
+
+# the tests must be run as root
+RESTRICT="test"
+
+# sys-apps/util-linux is required for libuuid
+RDEPEND="
+	acct-group/gluster
+	acct-user/gluster
+	dev-libs/libaio
+	dev-libs/userspace-rcu:=
+	net-libs/rpcsvc-proto
+	sys-apps/util-linux
+	sys-libs/readline:=
+	emacs? ( >=app-editors/emacs-23.1:* )
+	fuse? ( >=sys-fs/fuse-2.7.0:0 )
+	georeplication? ( ${PYTHON_DEPS} )
+	infiniband? ( sys-fabric/libibverbs:* sys-fabric/librdmacm:* )
+	xml? ( dev-libs/libxml2 )
+	!elibc_glibc? ( sys-libs/argp-standalone )
+	libtirpc? ( net-libs/libtirpc:= )
+	!libtirpc? ( elibc_glibc? ( sys-libs/glibc[rpc(-)] ) )
+	!libressl? ( dev-libs/openssl:=[-bindist] )
+	libressl? ( dev-libs/libressl:= )
+"
+DEPEND="
+	${RDEPEND}
+	sys-devel/bison
+	sys-devel/flex
+	virtual/acl
+	test? ( >=dev-util/cmocka-1.0.1
+		app-benchmarks/dbench
+		dev-vcs/git
+		net-fs/nfs-utils
+		virtual/perl-Test-Harness
+		dev-libs/yajl
+		sys-fs/xfsprogs
+		sys-apps/attr )
```

Comment:
Also please sort these test deps alphabetically when you have time / are touching these ebuilds next time.